### DISCUSSION TITLE
welcome View access for Target Extesnion

### DISCUSSION THREAD

**Discussion by swathisuja** [redhat-developer/vscode-extension-tester#1284] [Q&A]
HI team,

I am hosting a vs code instance with target extension to validate it. I am not able to access its welcome view content (button specifically) using welcomecontent page object. Can you help to access the welcome view of the Target extension ? 

Thanks,
Swathi

**Comment by djelinek** [comment]
Hello,
am I able to somewhere look on "the Target" extension? or at elast could you provide the screenshot how the welcome view looks like?

**Reply by swathisuja** [reply]
Hi,

Currently its in development stage and confidential content. I would be able to share it.  It contains a button with text Install ***** . 

Something similar to this
![image](https://github.com/redhat-developer/vscode-extension-tester/assets/28916423/5e959df5-9253-43b9-a03d-7e2459d3dee9)
 
I was able to get handle with help of  new WelcomeContentSection(panel, sectionparent).getButtons(); 
. 
but panel element and section parent xpaths are not reliable and fails with error Unknow Locator . 

Any other option to handle Welcome view buttons?

Thanks

**Reply by djelinek** [reply]
Can you share with me a code which is not working for you?

you can also take a look into our tests which are doing something similar I believe - https://github.com/redhat-developer/vscode-extension-tester/blob/main/tests/test-project/src/test/xsideBar/customView.test.ts#L123

Let me know if you will need more assistance
